miners/text/sample_config.txt

```diff
@@ -14,14 +14,14 @@ dendrite.max_worker_threads: 150
 dendrite.requires_grad: true
 dendrite.timeout: 5
 
-logging.debug: false
+logging.debug: true
 logging.logging_dir: ~/.bittensor/miners/
 logging.record_log: false
 logging.trace: false
 
 miner.device: cpu
-miner.epoch_length: 500
-miner.learning_rate: 0.03
+miner.epoch_length: 100
+miner.learning_rate: 1
```

Comment:
learning rate 1? isn't that epically high? :D

Reply:
yeap, it is extremely high. I actually tuned it down from 5 in the original PyTorch example